code/game/turfs/simulated/floor/chasm.dm

```diff
@@ -198,20 +208,73 @@
 	if(!AM || QDELETED(AM))
 		return
 
-	if(isrobot(AM))
-		var/mob/living/silicon/robot/S = AM
-		qdel(S.mmi)
-
 	falling_atoms -= AM
-
-	qdel(AM)
-
-	if(AM && !QDELETED(AM))	//It's indestructible
-		visible_message("<span class='boldwarning'>[src] spits out [AM]!</span>")
+	if(isliving(AM))
 		AM.alpha = oldalpha
 		AM.color = oldcolor
 		AM.transform = oldtransform
-		AM.throw_at(get_edge_target_turf(src,pick(GLOB.alldirs)),rand(1, 10),rand(1, 10))
+		var/mob/living/fallen_mob = AM
+		fallen_mob.notransform = FALSE
+		if(fallen_mob.stat != DEAD)
+			fallen_mob.death()
+			fallen_mob.adjustBruteLoss(1000) //crunch from long fall, want it to be like legion in damage
```

Comment:
While it is probably fine, wouldn't 1000 brute damage already instantly kill the person?

Reply:
an elite theoretically would die exactly from 1000 damage, though if we even slightly increased an elites health or gave it a brute mod it would survive

Reply:
Fair enough, you can never be too sure with those damned admins :p